src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,125 @@
+
+The annually elected EuroPython Society (EPS) Board manages the day-to-day operations of the EPS, the legal entity behind the EuroPython conference series. The EPS 2024 Board were responsible for most of the fiscal, legal and venue related work for the EuroPython 2024 conference, including venue & supplier selections and contract negotiations.
+
+- Alexander Hendorf
+- Anders Hammarquist (Treasurer)
+- Artur Czepiel (Chair)
+- Artem Kislovskiy (Vice-Chair)
+- Laís Carvalho
+- Sangarshanan Veera
+- Sebastiaan Zeeff
+- Theofanis Petkos
+- Vaibhav (VB) Srivastav
+
+
+# Code of Conduct Committee
+
+These humans are responsible for receiving & handling Code of Conduct reports and help the EuroPython community remain a safe and friendly place:
+
+- Naomi Ceder
+- Cheuk Ting Ho
+- Lumír Balhar
+- Jakub Vysoky
+
+# Communications & Design
+
+As you browse the clear & witty content on our website, blog, newsletter, and social media, you'll be enjoying the work of our talented wordsmiths and meme creators from the Communications team.
+
+- Laís Carvalho
+- Jodie Burchell
+- Joana Owusu-Appiah
+- Sena Şahin
+- Sangarshanan Veera
+
+The small and big design stuff you saw including t-shirts, badges, banners, stickers and social cards were scrambled together with love by our Design team:
+
+- Panagiotis
+- Laís Carvalho
+- Patrick Arminio
+
+
+# Financial Aid team
+
+If you are one of the 221 applicants of EuroPython 2024 finaid programme then you have surely exchanged many emails with some of our Finaid team members. From spreadsheet wizardry, to responding to countless emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
+
+- Alessia Marcolini
+- Cheuk Ting Ho
+- Vicky Twomey-Lee
+
+
+# Programme Team
+
+The folks managing Call for Proposals (CFP), community voting and the entire programme selection & scheduling are:
+
+- Artem Kislovskiy
+- Artur Czepiel
+- Cristián Maureira-Fredes
+- Cyril Bitterich
+- Diego Russo
+- Javier Lim
+- Jimena
+- Karolina
+- Mariajose Molina
+- Mia Bajić
+- Rodrigo Girao Serrao
+- Theofanis Petkos
+- Vaibhav Srivastav (VB)
+- Valerio Maggio
+- Yuliia Barabash
+
+
+# Operations Team
+
+From managing recording & streaming, to liaising with our wonderful remote and on-site volunteers, the Operations team have all covered. If you ever received a reply from our helpdesk, chances are you interacted with someone from this team :)
+
+- Anežka Müller
+- Angel Ramboi
+- Aris Nivorlis
+- Artem Kislovskiy
+- Cyril Bitterich
+- Daksh P. Jain
+- Ege Akman
+- George
+- Nicolás Demarchi
+- Jakub Červinka
+- Liberta Gani
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Panagiotis
+- Piotr Gnus
+- Raquel Dou
+- Saurav
+- Sebastiaan Zeeff
+- Shekhar Koirala
+- Theofanis Petkos
+
+
+# Sponsors Team
+
+All the sponsorship contacts, logistics & the exhibits are managed by the small team:
+
+- Angel Ramboi
+- Artem Kislovskiy
+- Artur Czepiel
+- Konstantin
+- Naa Ashiorkor Nortey
+
+
```

Comment:
Social Event organisers?

Reply:
Yes, all the special event organisers are pending cause i don't know all their names, need to ping the main organisers for it

Reply:
Should we try to add social event organisers we know in or you've got a list somewhere? :blush:

Reply:
I have added the names for the social event! i think id need help with the other events